### DISCUSSION TITLE
Collapsable Header Menu on mobile device

### DISCUSSION THREAD

**Discussion by sazzer** [fomantic/Fomantic-UI#1173] [Collections (form, grid, menu, message, table, ...)] [answered]
# Help Wanted

## Problem
I'm trying to build a menu that renders in its entirety on a large screen, but when using a small screen will instead render a collapsed menu that can be expanded on demand.

I *really* hate to compare to other systems, but the equivalent of the Bootstrap NavBar Toggler (https://getbootstrap.com/docs/4.3/components/navbar/#toggler).

I've seen mention of using the Sidebar module for this, but I can't see any obvious way of doing this only on certain device sizes - instead it seems to be all or nothing.

Cheers

**Comment by y0hami** [comment, marked as answer]
@sazzer This is fairly simple to achieve, take a look at this JSFiddle https://jsfiddle.net/nzwqg08b/ 😄

**Comment by y0hami** [comment]
I'm going to close this as the fiddle provided fulfills the question.

@sazzer If you need anymore help let us know and we can reopen this.
